frontend/app/assets/stylesheets/spree/frontend/variables/variables.scss

```diff
@@ -1,3 +1,4 @@
+$footer-background: #ffffff;
```

Comment:
@AleksandraKaminska default value of footer background should be `$primary-background`